### PR TITLE
Add and Remove

### PR DESCRIPTION
**Pull request by fherrerao**
# Hi There 💯 

- JavaScript file for the new functionality.
- A function for adding a new task (add a new element to the array).
- A function for deleting a task (remove an element from the array).
- A function for editing task descriptions.
- All changes are saved in local storage.

### CODE REVIEW

File: src/functions.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        const items = document.createElement('li');
+        items.classList.add('item-list');
+        items.id = element.index - 1;
+        items.innerHTML = `
+              <input type="checkbox" id="id-${element.index}">        
+              <p class="label" contenteditable="true">${element.description}</p>
+              <box-icon class="icon" name='dots-vertical-rounded'></box-icon>
+              <box-icon class="trash-icon d-none" name='trash'></box-icon>`;
+
+        itemsContainer.appendChild(items);
```
Comment (brytebee) [inline src/functions.js]: - [x] **Required:** You have correctly implemented the add event using the ENTER key press. 💪 However, you need an icon on which to implement a click event to add an item to the list.

Your Design | Sample template
:---: | :---|
image | image

- [x] **Required:** Users should be able to delete items on the to-do list. This, like the stated add button above, also required an element to fire the event. At the moment the trash-icon specified in your code does not display either on item creation on when checked.
image 
Please let's fix this bug. 🐛

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @fherrerao👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- fantastic use of keydown/keypress event✔️
- descriptive commit messages✔️
- good GitHub flow✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **OPTIONAL:** Your live demo link opens to your README Github page, it would be nice to fix it. right ❓ 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/functions.js (hunk 1 above)
Comment (fherrerao) [reply src/functions.js]: Hi @brytebee, thanks for your feedback, kindly the checkbox functionality is not a requirement for this project, maybe for the next 🐶 , but when you click on the name of the task will display the trash-icon. 😃 

todo

File: src/functions.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    let indexArray;
+    Collection.tasks.forEach((element, index) => {
+      if (element.description === del) {
+        indexArray = index;
+      }
+    });
+    Collection.tasks.splice(indexArray, 1);
+    Collection.setLocalStorage();
+    window.location.reload();
+  };
```
Comment (MrBrN197) [inline src/functions.js:47]: - [x] [OPTIONAL] As part of the requirement we need the indexes to be updated on localStorage when you delete an item. if you don't do this you'll end up in a situation where you have multiple tasks with the same index. 

image

**Review by MrBrN197 — APPROVED** [review]
## Hi, @fherrerao 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**